_layouts/blog-post.html

```diff
@@ -7,11 +7,15 @@
     <h2 class="text-center mt-5">{{ page.title }}</h2>
     <p class="text-center">{{ page.date | date: "%d %B %Y" }}, by {{ page.author }}</p>
   </div>
-  <div class="row">
-    <div class="col-lg-8 mx-auto">
-      <img class="img-fluid my-4" src="{{ site.baseurl }}/img/posts/{{ page.image }}" alt="{{ page.image }}">
-      <p class="lead">{{ page.intro}}</p>
-      {{ content }}
-    </div>
+  <div class="col-lg-8 mx-auto">
+    <img class="img-fluid my-4" src="{{ site.baseurl }}/img/posts/{{ page.image }}" alt="{{ page.image }}">
+    <p class="lead">{{ page.intro}}</p>
+    {{ content }}
+  </div>
+  <div class="col-lg-8 mx-auto">
+    <span class="pr-3">Share</span>
+    <span class="pr-3"><a href="https://twitter.com/home?status=https%3A//laws.africa/{{ page.url }}"><img src="{{ site.baseurl }}/img/social/twitter.svg" /></a></span>
```

Comment:
For the links, page.url already starts with a /, so `laws.africa/` -> `laws.africa`, and `{{ site.baseurl }}` can be remove.

Reply:
Removed